WordPress/gutenberg
# Data views: Display all checkboxes when a single item is selected

## What?
Display all checkboxes in data views table layout as soon as a single item is selected.

## Why?
Currently, checkboxes are only visible on hover, focus, and when checked. This can result in views where some items appear to have a checkboxes and others do not. At a glance this might be considered confusing.

<img width="1509" alt="Screenshot 2024-02-21 at 09 57 59" src="https://github.com/WordPress/gutenberg/assets/846565/925d8783-9e09-4f77-8717-4f6d2dec77fb">


This PR updates the logic so that _all_ checkboxes become visible when any item is selected. 

## How?

`.dataviews-view-table:has(tr.is-selected) .components-checkbox-control__input { opacity: 1 }`

## Testing Instructions
1. Open a data view in the site editor e.g. "Manage all pages".
2. Select a single row.
3. Observe that checkboxes appear for all rows.



https://github.com/WordPress/gutenberg/assets/846565/05fa2240-2ef0-45c1-b63f-7e4aa22951c5
